### DISCUSSION TITLE
Welcome to AbpHelper.CLI Discussions!

### DISCUSSION THREAD

**Discussion by gdlcf88** [EasyAbp/AbpHelper.CLI#194] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Katoji-Rikito** [comment]
Hi everyone, how to select DbContext in "abphelper generate"? Since my project has at least 2 DbContext.
